### PR TITLE
Add New Cards Logic and UI

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Add new the 3 new statistics cards
Unit an UITests will be created in a separate PR

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7086

## Video
https://user-images.githubusercontent.com/15270737/121205409-f14dc000-c877-11eb-979e-b5331de1b4df.MP4

### CODE REVIEW

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient+Configuration.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ extension HTTPClient {
 		static func makeDefaultConfiguration(environmentProvider: EnvironmentProviding) -> Configuration {
 			let endpoints = Configuration.Endpoints(
 				distribution: .init(
-					baseURL: environmentProvider.currentEnvironment().distributionURL,
+					baseURL: URL(string: "https://cwa-mock.eu.ngrok.io")!,
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient+Configuration.swift:15]: Shouldn't be checked in

File: src/xcode/ENA/ENA/Source/Client/HTTP Client/Verification.swift [hunk 2]
```diff
@@ -23,22 +23,23 @@ struct SignatureVerifier: SignatureVerification {
...
+//					return true
+//				}
+//			}
+//
+//			return false // no match at all
+//		} catch {
+//			Log.error("Package verification error! \(error.localizedDescription)", log: .crypto, error: error)
+//			return false
+//		}
+	return true
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Client/HTTP Client/Verification.swift:42]: Same here

**Review by naveeddotio — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Client/HTTP Client/Verification.swift (hunk 2 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Client/HTTP Client/Verification.swift:42]: As mentioned this is not finished i just pushed to check the functionality, so anyone can run the branch and get the new cards

**Pull request closed without merging**